docs/limit-login-attempts.md

```diff
@@ -0,0 +1,95 @@
+# Limit Login Attempts
+
+To protect against brute force attacks, Altis has the ability to limit the number of login attempts possible both through normal login as well as using auth cookies.
+
+This feature blocks an IP address from making further attempts after a specified limit on retries is reached, making a brute-force attack difficult or impossible.
+
+- Limit for the number of retry attempts when logging in (for each IP).
+- Limit the number of attempts to log in using auth cookies in same way.
+- Informs user about remaining retries or lockout time on login page.
+- Optional logging, optional email notification.
+
+**Note:** In this document, we'll only show the `limit-login-attempts` configuration for brevity.
+
+To set the limit login attempts security settings, set values in your `composer.json` configuration under `extra.altis.modules.security.limit-login-attempts`. The default values are below, but all can be overriden.
+
+```json
+{
+	"extra": {
+		"altis": {
+			"modules": {
+				"security": {
+					"limit-login-attempts": {
+						"allowed_retries": 4,      // Lock out after this many tries.
+						"lockout_duration": 1200,  // Lock out for this many seconds - default to 20 minutes.
+						"allowed_lockouts": 4,     // Long lock out after this many lockouts.
+						"long_duration": 86400,    // Long lock out for this many seconds - defaults to 24 hours.
+						"valid_duration": 43200,   // Reset failed attempts after this many seconds - defaults to 12 hours.
+						"cookies": 1,              // Also limit malformed/forged cookies?
+						"lockout_notify": "log",   // Notify on lockout. Values: '', 'log', 'email', 'log,email'.
+						"notify_email_after": 4,   // If notify by email, do so after this number of lockouts.
+						"lockout_method": "ip"     // Method to use for lockout.
+					}
+				}
+			}
+		}
+	}
+}
+```
+## Configuration settings
+
+By setting the configuration settings in your `composer.json`, we lock the settings to those values and hide the admin settings page.
```

Comment:
```suggestion
By placing the configuration settings in your `composer.json`, the settings are locked to those values and the admin settings page is hidden.
```

This change is just about the use of "we". I think that's been done in some places in the docs but I'd prefer to avoid it and talk from the perspective of what Altis does rather than "us" if that makes sense.